src/main/java/com/google/sps/ProductDetectionAPI.java

```diff
@@ -0,0 +1,24 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps;
+
+import com.google.sps.data.ProductDetectionData;
+
+/**
+ * Interface for ProductDetectionAPIImpl, respective FakeProductDetectionAPI, used for testing.
```

Comment:
```suggestion
 * Interface used for functionality related to Cloud Vision API.
```

Reply:
As we are using its implementations not only for testing but also in production code :)

Reply:
Oh, I meant that the latter is used for testing, obviously :) But now that I see there can be misunderstandings, I'll update it